docs/messaging-service/email-config.md

```diff
@@ -0,0 +1,32 @@
+# Email gateway configuration
+
+The Messaging service supports 3 ways of sending email messages out of the box:
+
+- By integrating with the OneWelcome email gateway
+- By integrating with an SMTP server
+- By integrating with a custom API
+
+## OneWelcome email gateway integration configuration
+
+The OneWelcome email gateway is the default integration, as all necessary elements are hosted by OneWelcome. To use this email gateway, you need to specify the
+sender email address, sender name and reply-to address. Be aware that OneWelcome only allows _verified_ sender email addresses and domains.
```

Comment:
unnecessary line break. should be continuing the sentence above.

Reply:
https://github.com/onewelcome/onegini-identity-cloud-documentation/pull/99#discussion_r762764856